app.js

```diff
@@ -184,6 +190,7 @@ var spOptions = {
   authnRequestBinding:          'HTTP-Redirect',
   validateInResponseTo:         true,
   privateCert:          argv.spPrivateKey,
+  additionalParams:{'RelayState':argv.relayState},
```

Comment:
Looked like the way to pass in the relay state was through additionalParams into the SAML strategy. I'm not too familiar with this so hopefully it is correct.